docs/theming/developers.md

```diff
@@ -200,41 +200,41 @@ themable graphics
 
 ## How theming works
 
-We developed our theming system with web standards in mind, aiming for it to use
-CSS over JavaScript as much as possible. At the current state of the art, we
-found that some JavaScript is required for the system to work correctly, but in 
-the near  future, we expect to be able to remove all or nearly all of that 
-JavaScript, and have the theming system work entirely (or almost entirely) 
+We developed our theming system with web standards in mind, aiming to use
+CSS over JavaScript as much as possible. At the current state of browser technology,
+we found that some JavaScript is required for the system to work correctly,
+but shortly, we expect to be able to remove all or nearly all of that 
+JavaScript and have the theming system work entirely (or almost entirely) 
 through plain CSS. The following explains how the system currently operates, how
 element authors can hook into it, and what the future holds for theming.
 
 ### Current implementation - Context Protocol
 
-<abbr title="Red Hat Design System">RHDS</abbr>' Theming system is primarily 
-about styles. Tt currently relies on JavaScript to work, by way of the [context 
+<abbr title="Red Hat Design System">RHDS</abbr>’ Theming system is primarily 
+about styles. It currently relies on JavaScript to work, via the [context 
 protocol][contextprotocol] developed by the web components community to support 
 passing data between components.
 
 Our system utilizes this protocol with the setting of the `color-palette` 
 attribute on a provider element which makes its context data (in our case, 
-`light` or `dark`) to it's children.  By doing so we can ensure accessible 
+`light` or `dark`) to its children. By doing so we can ensure accessible 
 colors are applied given any possible change in context value higher up in the 
 <abbr title="Document Object Model">DOM</abbr> tree. 
 
-This is important not only for helping us maintain our own design guidelines, 
-but also for accessibility compliance and enabling great experiences for all our 
+This is important not only for helping us maintain our design guidelines but
+also for accessibility compliance and enabling great experiences for all our 
 users.
 
 The context protocol is enabled by two [reactive controllers][controllers]: the
 [provider controller][providersrc], and the [consumer controller][consumersrc].
-These controllers work in tandem to alternately provide the context data, or
+These controllers work together to provide the context data, or
```

Comment:
I wasn't sure about "together"  can we really say they "work together to provide and consume"? One provides, the other consumes. They work together to propagate styles to the correct elements, but not to either provide or consume

Reply:
Good call. Check out [1bdecba](https://github.com/RedHat-UX/red-hat-design-system/pull/1937/commits/1bdecba001a4bc4752245dc512bb6858eaac781c) and see if that is an improvement.